testing/tests/DevExpress.ui.widgets.editors/rangeSlider.tests.js

```diff
@@ -1343,3 +1343,81 @@ QUnit.module('valueChanged handler should receive correct event parameter', {
         });
     });
 });
+
+QUnit.module('valueChangeMode option', {
+    beforeEach: function() {
+        this.valueChangedHandler = sinon.stub();
+        this.$element = $('#slider').dxRangeSlider({
+            max: 100,
+            min: 0,
+            start: 20,
+            end: 60,
+            valueChangeMode: 'eventual',
+            onValueChanged: this.valueChangedHandler,
+            tooltip: {
+                enabled: true,
+                showMode: 'always'
+            }
+        });
+        this.instance = this.$element.dxRangeSlider('instance');
+        this.$handle = this.$element.find(`.${SLIDER_HANDLE_CLASS}`);
+        this.leftHandle = this.$handle.eq(0);
+        this.rightHandle = this.$handle.eq(1);
+        this.leftHandleTooltip = this.leftHandle.find(`.${TOOLTIP_CLASS}`);
+        this.rightHandleTooltip = this.rightHandle.find(`.${TOOLTIP_CLASS}`);
+
+        this.range = this.$element.find('.' + SLIDER_RANGE_CLASS);
+
+        this.getLeftTooltipText = () => $.trim(this.leftHandleTooltip.text());
+        this.getRightTooltipText = () => $.trim(this.rightHandleTooltip.text());
+    }
+}, () => {
+    QUnit.test('left slider handle value should not change on swipe with "eventual" valueChangeMode', function(assert) {
```

Comment:
there is no such thing like left slider handle value, there is only option value, so let's name it something like "value option should not change on left handle swipe with "eventual" valueChangeMode'